PocketKit/Sources/Sync/Requests.swift

```diff
@@ -92,7 +92,7 @@ public enum Requests {
 
     public static func fetchRecomendations(by lineupIdentifier: String) -> RichFetchRequest<Recommendation> {
         let request = RichFetchRequest<Recommendation>(entityName: "Recommendation")
-        request.predicate = NSPredicate(format: "slate.slateLineup.remoteID = %@ AND item != NULL", lineupIdentifier)
```

Comment:
Can we add a comment about how there should only ever be 1 lineup for now, or something?

Reply:
done